test/javascripts/acceptance/keyboard-shortcuts-test.js.es6

```diff
@@ -0,0 +1,85 @@
+import { acceptance } from "helpers/qunit-helpers";
+import DiscourseURL from "discourse/lib/url";
+
+acceptance("Keyboard Shortcuts", { loggedIn: true });
+
+test("go to first suggested topic", async assert => {
+  sandbox.stub(DiscourseURL, "routeTo");
+
+  /*
+   * No suggested topics exist.
+   */
+
+  server.get("/t/9/last.json", () => [
+    200,
+    { "Content-Type": "application/json" },
+    {}
+  ]);
+
+  await visit("/t/this-is-a-test-topic/9");
+  await keyEvent(document, "keypress", "g".charCodeAt(0));
+  await keyEvent(document, "keypress", "s".charCodeAt(0));
+  assert.ok(DiscourseURL.routeTo.getCalls().length == 0);
+
+  /*
+   * Suggested topics elements exist.
+   */
+
+  await visit("/t/internationalization-localization/280");
+  await keyEvent(document, "keypress", "g".charCodeAt(0));
+  await keyEvent(document, "keypress", "s".charCodeAt(0));
+  assert.ok(
+    DiscourseURL.routeTo.calledWith("/t/polls-are-still-very-buggy/27331/4")
+  );
+
+  /*
+   * Suggested topic is returned by server.
+   */
+
+  server.get("/t/28830/last.json", () => [
+    200,
+    { "Content-Type": "application/json" },
+    {
+      suggested_topics: [
```

Comment:
Do we need all these data for the purpose of this test?

Reply:
Nope, just the `id` and `slug`.